src/scandeval/benchmark_config_factory.py

```diff
@@ -155,17 +155,18 @@ def prepare_languages(language: str | list[str]) -> list[str]:
     return languages
 
 
-def prepare_model_languages(
-    model_language: str | list[str] | None,
-    languages: list[str],
+def prepare_languages(
+    language: str | list[str] | None,
+    language_codes: list[str],
```

Comment:
Maybe rename these to `language_codes` and `default_language_codes` or something like that?